src/deploy.js

```diff
@@ -1,15 +1,85 @@
-import { spawn } from './utils';
+import AWS from 'aws-sdk';
+import s3 from 's3-node-client';
+import dotenv from 'dotenv';
+
+const path = require('path');
+
+// default build directory
+const BUILD = 'build/';
+
+/**
+ * Returns an object with all variables loaded from a environment
+ * @param {string} environmentName is the suffix after .env.*
+ */
 
 const deploy = async (opts) => {
-  const { path } = opts;
-
-  console.error(`path passed '${path}'`);
-  console.log(process.cwd());
-  try {
-    await spawn('./src/test.sh');
-  } catch (e) {
-    console.error(e);
-  }
+  // const { path } = opts;
+
+  const usageMessage = console.log(
+    'usage: $0 [-e <path/to/file>] [-v <version string>] [-b <path/to/build>]',
+  );
+
+  console.log(`Exectued with path: ${opts.path}`);
```

Comment:
Typo in `executed`. Also, we use all small caps in logs. I would suggest:

`deploying app in {...} directory`

or something similar.